tests/conftest.py

```diff
@@ -14,12 +14,14 @@
 from aad_fastapi.roles.role_requirement import RoleRequirement
 
 os.environ["CLIENT_ID"] = "01010101-aaaa-bbbb-acdf-020202020202"
+os.environ["CLIENT_SECRET"] = "a_secret_from_azure_ad"
```

Comment:
is this used?

Reply:
oh, no, was just a test on optional things